owncloud/setup.cfg

```
@@ -0,0 +1,30 @@
+[metadata]
+name = indico-plugin-owncloud
+version = 3.2
```

```suggestion
version = 3.2-dev
```

sorry, forgot that we only bump the version from dev to a normal one once we make a proper release; otherwise building the wheels with the `+date` suffix will cause higher versions than the stable 3.2 one